src/heretic/p_spec.c

```diff
@@ -886,8 +890,20 @@ void P_PlayerInSpecialSector(player_t * player)
                 P_HitFloor(player->mo);
             }
             break;
+        // Set flag when secret incrmented so message displayed?
```

Comment:
Erm, sorry?

Ths line isn't really helpful. Please consider removing it or turning it into a properly worded `[crispy]` comment.